<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyProfile</class>
 <widget class="QWidget" name="ModifyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>542</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>542</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>659</width>
    <height>542</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modifica Profilo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #0082b6;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica il tuo Profilo</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>30</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="personalDataGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Dati Anagrafici</string>
   </property>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="surnameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Cognome</string>
    </property>
   </widget>
   <widget class="QLabel" name="passwordLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="contactGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;
</string>
   </property>
   <property name="title">
    <string>Contatti</string>
   </property>
   <widget class="QLabel" name="addressLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Indirizzo</string>
    </property>
   </widget>
   <widget class="QLabel" name="telephoneLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>59</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Telefono</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>59</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addressLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="telephoneLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>54</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="experienceTextEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid;
border-radius: 10px 10px 10px 10px;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="qualificationTextEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid;
border-radius: 10px 10px 10px 10px;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="personalTextEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>221</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
border: 1px solid;
border-radius: 10px 10px 10px 10px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="experienceLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Inserisci nuova esperienza lavorativa</string>
   </property>
  </widget>
  <widget class="QLabel" name="qualificationLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>195</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Inserisci nuova esperienza didattica</string>
   </property>
  </widget>
  <widget class="QLabel" name="personalLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>345</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Inserisci nuova esperienza personale</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="languagesGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>331</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(78, 155, 209);
border-radius: 10px 10px 10px 10px;
border: 1px solid black;
color:white;</string>
   </property>
   <property name="title">
    <string>Lingue Conosciute</string>
   </property>
   <widget class="QCheckBox" name="italianLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Italiano</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="englishLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Inglese</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="portogheseLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Portoghese</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="spanishLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Spagnolo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="frenchLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Francese</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="germanLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Tedesco</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="japaneseLanguageCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Giapponese</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="otherLanguagesTextLine">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>115</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
background-color: white;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="otherLanguagesCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string>Altro</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="experienceCheckBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="qualificationCheckBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="personalExperienceCheckBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>410</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
